src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/inventory/inventory.component.ts

```diff
@@ -61,12 +68,16 @@ export class InventoryComponent implements OnChanges, OnInit {
       {
         name: this.i18n('Rotates'),
         prop: 'sys_api.rotational',
-        flexGrow: 1
+        flexGrow: 1,
+        cellClass: 'text-center',
+        cellTransformation: CellTemplate.checkIcon
       },
       {
         name: this.i18n('Available'),
         prop: 'available',
-        flexGrow: 1
+        flexGrow: 1,
+        cellClass: 'text-center',
+        cellTransformation: CellTemplate.checkIcon
       },
```

Comment:
I think that instead of displaying some columns, like this one, we could used them for altering the row format (I think we already discussed this in the past): for example, [as disabled rows](https://semantic-ui.com/collections/table.html#disabled).

Reply:
IMO this should be done in a separate PR because other datatable might be affected by this, too. Maybe you should open a tracker issue for this.

Reply:
Done: https://tracker.ceph.com/issues/42962